src/utils/batteryParameters.js

```diff
@@ -0,0 +1,65 @@
+const parameters = [
+  {
+    name: "Temperature",
+    value: 25,
+    lowerBound: 0,
+    upperBound: 45,
+    lowMessage: "too low",
+    highMessage: "too high",
+  },
+  {
+    name: "State of Charge",
+    value: 50,
+    lowerBound: 20,
+    upperBound: 80,
+    lowMessage: "too low",
+    highMessage: "too high",
+  },
+  {
+    name: "Charge Rate",
+    value: 0.5,
+    lowerBound: null, //because chargeRate has no lower bound
+    upperBound: 0.8,
+    lowMessage: "too low",
+    highMessage: "too high",
+  },
+];
```

Comment:
Sir, in the future, if new parameters need to be added, they can be included in this array of objects, without increasing the complexity.